test/integration/test_utils.js

```diff
@@ -73,3 +73,19 @@ function getComputedStyleSelector(id) {
   return `getComputedStyle(${getQuerySelector(id)})`;
 }
 exports.getComputedStyleSelector = getComputedStyleSelector;
+
+const editorPrefix = "#pdfjs_internal_editor_";
+exports.editorPrefix = editorPrefix;
+
+function getSelectedEditors(page) {
+  return page.evaluate(pref => {
+    const elements = document.querySelectorAll(".selectedEditor");
+    const results = [];
+    for (const element of elements) {
+      results.push(parseInt(element.id.slice(pref.length)));
```

Comment:
Nit: This could maybe be shortened a tiny bit?
```suggestion
    for (const { id } of elements) {
      results.push(parseInt(id.slice(pref.length)));
```